index.bs

```diff
@@ -843,4 +1380,227 @@ result = (
   unknown-error: 199
 )
 
+; type key 17
+remote-playback-availability-request = {
+  request
+  1: [* text] ; urls
+  2: microseconds ; watch-duration
+  3: int ; watch-id
```

Comment:
Seems be better to use `uint`instead of `int` for consistency with `connection-id`, `request-id`

Reply:
This comment and your others apply to Peter's PR (#134).  Unfortunately, it seems like GitHub doesn't have support for dependent PRs (or maybe just not from separate authors?).  Looking at the individual commits is easier for me than looking at the total diff.